src/coreComponents/dataRepository/KeyNames.hpp

```diff
@@ -30,41 +30,11 @@ namespace keys
 
 /// @cond DO_NOT_DOCUMENT
 
-//static constexpr auto ReferencePosition = "ReferencePosition";
-static constexpr auto referencePositionString = "ReferencePosition";
-
-static constexpr auto TotalDisplacement = "TotalDisplacement";
-static constexpr auto IncrementalDisplacement = "IncrementalDisplacement";
-static constexpr auto Velocity = "Velocity";
-static constexpr auto Acceleration = "Acceleration";
-static constexpr auto Mass = "Mass";
-static constexpr auto Force = "Force";
-static constexpr auto Strain = "Strain";
```

Comment:
So great to see all these key vanish!
This is awesome!

Reply:
I struggled to remove this file entirely because of two remaining keys:
- `ProblemManager` -> actually used in the `dataRepository` (`xmlWrapper`, `WrapperBase`, `Group` class)
- `cellManager` -> used in two distant parts of the code (`problemManager` and the mesh classes)  

so for now, I kept the file `KeyNames.hpp` in the code